index.js

```diff
@@ -103,7 +104,9 @@ module.exports = fp(function from (fastify, opts, next) {
       if (err) {
         this.request.log.warn(err, 'response errored')
         if (!this.sent) {
-          if (err.code === 'ERR_HTTP2_STREAM_CANCEL') {
+          if (onError) {
+            onError(this.request.req, this, err)
```

Comment:
I believe `this` will be an instance of Fastify's reply object. Thus, `this.request` should be passed instead of `this.request.req` so that a Fastify request object is passed.

Reply:
`this.request.req` is Fastify's reply object, i also wanted to keep it aligned with `onResponse`.

Reply:
Oh. These are from the `request` library.